datarobot_batch_scoring/batch_scoring.py

```diff
@@ -772,7 +807,8 @@ def run_batch_predictions(base_url, base_headers, user, pwd,
                           out_file, keep_cols, delimiter,
                           dataset, pred_name,
                           timeout, ui, fast_mode, auto_sample,
-                          dry_run, encoding, skip_dialect):
+                          dry_run, encoding, skip_dialect,
+                          max_batch_size=3.5 * 1024 ** 2):
```

Comment:
I agree that this doesn't need to be another argument to the main function.  There are enough of those already.
My initial comment was more aimed at the fact that we are setting max size to "3.5 * 1024 ** 2" instead of "10 * 1024 ** 2". I'm confused by that.